runtime/sema/check_member_expression.go

```diff
@@ -301,80 +305,84 @@ func (checker *Checker) visitMember(expression *ast.MemberExpression) (accessedT
 				},
 			)
 		}
-	} else {
 
-		if checker.PositionInfo != nil {
-			checker.PositionInfo.recordMemberOccurrence(
-				accessedType,
-				identifier,
-				identifierStartPosition,
-				identifierEndPosition,
-			)
-		}
+		return
+	}
 
-		// Check access and report if inaccessible
-		accessRange := func() ast.Range { return ast.NewRangeFromPositioned(checker.memoryGauge, expression) }
-		isReadable, resultingAuthorization := checker.isReadableMember(accessedType, member, accessRange)
-		if !isReadable {
-			checker.report(
-				&InvalidAccessError{
-					Name:              member.Identifier.Identifier,
-					RestrictingAccess: member.Access,
-					DeclarationKind:   member.DeclarationKind,
-					Range:             accessRange(),
-				},
-			)
-		}
+	if checker.PositionInfo != nil {
```

Comment:
Here also, many of these changes are just refactoring. Check the diff without the whitespace, to see the actual logic changes.